client/signup/steps/theme-selection/index.jsx

```diff
@@ -30,28 +34,55 @@ module.exports = React.createClass( {
 		};
 	},
 
-	renderThemes: function() {
+	handleScreenshotClick: function( theme ) {
```

Comment:
I assume this code is from the old theme thumbnail component? Can we remove that at the same time?